src/wui/game_chat_panel.h

```diff
@@ -25,7 +25,7 @@
 #include "chat/chat.h"
 #include "ui_basic/editbox.h"
 #include "ui_basic/multilinetextarea.h"
-
+// TODO(jmoerschbach): replace with gamechatpanel2 once internetlobby is using boxlayout also
```

Comment:
I have an almost working branch for that - do you want to continue that branch for me?

https://github.com/gunchleoc/widelands/tree/internet-lobby-box

Reply:
yes, I will do that :)

Reply:
Thanks! I have just merged master.

The remaining issue with that branch is that the chatpanel blocks the ctrl+F screenshot for fullscreen switching.